pkg/apis/servicecatalog/validation/binding.go

```diff
@@ -23,13 +23,21 @@ import (
 	sc "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
 )
 
+// ValidateBindingName is the validation function for Binding names.
+var ValidateBindingName = apivalidation.NameIsDNSSubdomain
```

Comment:
I actually copied this from upstream - it seems unnecessarily restrictive - can I loosen in a followup?